bundles/automation/org.eclipse.smarthome.automation.providers/src/main/java/org/eclipse/smarthome/automation/internal/core/provider/AbstractResourceBundleProvider.java

```diff
@@ -302,8 +302,8 @@ protected void processAutomationProvider(Bundle bundle) {
         try {
             urlEnum = bundle.findEntries(path, null, false);
         } catch (IllegalStateException e) {
-            logger.debug("Can't read from resource of bundle with ID " + bundle.getBundleId()
-                    + ". The bundle is uninstalled.", e);
+            logger.debug(String.format("Can't read from resource of bundle with ID {}. The bundle is uninstalled.",
```

Comment:
why do you use String.format here? Formatting is directly done by the parameterized logging.